pkgs/development/libraries/science/math/nccl/tests.nix

```diff
@@ -46,12 +54,18 @@ cudaPackages.backendStdenv.mkDerivation (finalAttrs: {
     cp -r build/* $out/bin/
   '';
 
+  # Run the update script with: `nix-shell maintainers/scripts/update.nix --argstr package cudaPackages.nccl-tests`
```

Comment:
Is this documentation for yourself or intentional in the PR?  A quick search for it shows documentation in `doc/stdenv`.

Reply:
My bad, that was just for myself! Removed in latest.